devo/sender/lookup.py

```diff
@@ -81,7 +82,7 @@ def send_headers(self, headers=None, key="KEY", event='START',
         self.send_control(event=event, headers=p_headers, action=action)
 
     def send_data_line(self, key=None, fields=None,
-                       delete=False, key_index=None):
+                       delete=False, key_index=None, escape_quotes=False):
```

Comment:
I am unsure whether we should add additional parameters to the non-static methods or have an attribute that is defined in the constructor and used across all the possible static method calls. What do you think is better?